### DISCUSSION TITLE
Autocomplete not working

### DISCUSSION THREAD

**Discussion by EricZhang456** [searx/searx#3188] [Installation issue: custom] [answered]
I installed Searx using the official Nginx guide and set `autocomplete: "google"` in `/etc/searx/settings.yml`, however that didn't work. [opensearch.xml](https://searx.ericaftereric.top/opensearch.xml) has no mention of autocomplete as well.

Searx site: https://searx.ericaftereric.top

**Comment by unixfox** [comment, marked as answer]
Are you talking about the autocomplete of the browser or in the search bar?

**Reply by EricZhang456** [reply]
Both

**Reply by unixfox** [reply]
It works for me in the search bar. Did you try to clear your cookies?

**Reply by EricZhang456** [reply]
It works that way, thanks
